Atomic/diff.py

```diff
@@ -22,6 +22,10 @@ def cli(subparser):
                        help=_("Compare images' metadata"))
     diffp.add_argument("--json", default=False, action='store_true',
                        help=_("output json"))
+    diffp.add_argument("-k", "--keywords", nargs='+',
+                       choices=['link', 'nlink', 'mode', 'type', 'time', 'uid', 'gid', 'size', 'sha256digest'],
+                       default=['link', 'nlink', 'mode', 'type', 'time', 'uid', 'gid', 'size'],
+                       help=_("Exclusive keywords to be used for file level comparision"))
```

Comment:
@rhatdan I can do 'all' but if we ever add two algo's for digesting, they both would then get used.  Adding for now as it makes sense.